templates/custom/index.html

```diff
@@ -52,8 +93,21 @@ <h2>Your tests</h2>
 {% block scripts %}
     {{ super() }}
     <script type="text/javascript">
+        function toggle(source) {
+            var name = source.name + "-tr";
+            checkboxes = document.getElementsByName(name);
+            for(var i=0; i < checkboxes.length; i++)
+            {
+                checkbox = checkboxes[i].querySelectorAll('input')[0];
+                checkbox.checked = source.checked;
+            }
+        }
         $("input[name='commit_select']").change(function(){
             $("#commit_hash").val(this.value)
         });
+        $('.category-header').on('click', function(){
```

Comment:
Both this one and the one above aren't wrapped in a `$(document).ready()` call... Which could potentially cause javascript errors. Please do so.